pkg/oci/client/networking.go

```diff
@@ -105,3 +114,71 @@ func (c *client) UpdateSecurityList(ctx context.Context, request core.UpdateSecu
 func subnetCacheKeyFn(obj interface{}) (string, error) {
 	return *obj.(*core.Subnet).Id, nil
 }
+
+func (c *client) GetRouteTable(ctx context.Context, routeID string) (core.GetRouteTableResponse, error) {
+	resp, err := c.network.GetRouteTable(ctx, core.GetRouteTableRequest{
+		RtId: &routeID,
+	})
+	incRequestCounter(err, getVerb, routeTableResource)
+
+	if err != nil {
+		return core.GetRouteTableResponse{}, err
```

Comment:
These new methods don't match the others in that they return the response instead of the object itself and aren't returning *'s with nil when there is an error.